packages/seacas/scripts/cmake/Dependencies.cmake

```diff
@@ -1,9 +1,3 @@
-if(CMAKE_PROJECT_NAME STREQUAL "SEACAS")
-TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
-  LIB_REQUIRED_PACKAGES Exodus)
-)
```

Comment:
Again, why was this impacting the Trilinos build?

Reply:
Similar, wasn't impacting build, but was overly complicated so cleaned up.  I have updated main header comment on this PR to indicate that not all changes were directly related to Trilinos build.